tests/test_e2e_40_sdntrace.py

```diff
@@ -942,3 +942,50 @@ def test_080_validate_attribute_on_payload(self):
         api_url = KYTOS_API + '/amlight/sdntrace_cp/v1/traces'
         response = requests.put(api_url, json=payload)
         assert response.status_code == 200
+
+    def test_085_test_evcs_terminating_on_nnis(cls):
+        "Test EVCs terminating on NNIs"
+
+        cls.create_evc(999, "00:00:00:00:00:00:00:02:1", "00:00:00:00:00:00:00:04:1")
```

Comment:
Notice that this test suite still has a tech debit https://github.com/kytos-ng/kytos-end-to-end-tests/issues/239 of tearing down resources that are created during tests since it's only using `setup_class`. Here if you're going to create this EVC, make sure to also delete and assert that it gets deleted just so it doesn't increase tech debt (or lead to other surprises). As a general rule of thumb, any e2e test resources should not interfere on subsequent tests.

Reply:
In this PR, the created evcs are deleted in the test case. Probably, it will change when this test suits is refactored for [issue](https://github.com/kytos-ng/kytos-end-to-end-tests/issues/239).